### PR TITLE
UG: add edit commands

### PR DESCRIPTION
**Pull request by Thing1Thing2**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -124,7 +124,7 @@ Beyond individual products, you can also:
   e.g. in `addClient n/NAME ...`, `NAME` is a parameter which can be used as `addClient n/John Tan ...`.
 
 * Items in square brackets are optional, while those not in square brackets are compulsory.<br>
-  e.g `addClient n/NAME p/PHONE_NUMBER [e/EMAIL]` can be used as `addClient i/1 p/12345678 e/John@abc.com a/123 Avenue b/12122000` or as `addClient n/John Tan p/12345678`.
+  e.g `addClient n/NAME p/PHONE_NUMBER [e/EMAIL]` can be used as `addClient n/John Tan p/12345678 e/johntan@insurec.com` or as `addClient n/John Tan p/12345678`.
 
 * Parameters can be in any order.<br>
   e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
```
Comment (rexong) [inline docs/UserGuide.md:130]: I believe this one i done it already, hence there maybe conflicts that you need to resolve. My pr has already been merged. So you can use it to resolve the conflict.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -143,11 +143,12 @@ Beyond individual products, you can also:
...
 Adds a new client to MyInsuRec.
 
-Format: `addClient n/NAME p/PHONE_NUMBER [e/EMAIL] [b/BIRTHDAY] [pd/PRODUCT]`
+Format: `addClient n/NAME p/PHONE_NUMBER [a/ADDRESS] [e/EMAIL] [b/BIRTHDAY] [pd/PRODUCT]`
 
 * Adds a client having name `NAME`.
 * A client must have a NAME and a PHONE_NUMBER.
-* EMAIL and BIRTHDAY is optional.
+* ADDRESS, EMAIL, BIRTHDAY, and PRODUCT are optional.
+* PRODUCT should be created before adding to client.
```
Comment (rexong) [inline docs/UserGuide.md:152]: same here

File: docs/UserGuide.md [hunk 3]
```diff
@@ -189,20 +190,40 @@ Format: `delClient i/INDEX`
 Examples:
 * `delClient i/2`
 
+#### 3.1.5 Editing a client: `editClient`
+
+Edits details of the specified client.
+
+Format: `editClient i/INDEX [n/NAME] [p/PHONE_NUMBER] [a/ADDRESS] [e/EMAIL] [b/BIRTHDAY] [pd/PRODUCT]`
+
+* Edits the client at the specified `INDEX`.
```
Comment (rexong) [inline docs/UserGuide.md:199]: Should probably specify where the user can get the `INDEX` which in this case is from `listClient` command.

File: docs/UserGuide.md [hunk 4]
```diff
@@ -252,6 +273,22 @@ Format: `delMeeting i/INDEX`
 Examples:
 * `delMeeting i/2`
 
+#### 3.2.5 Editing a meeting: `editMeeting`
+
+Edits details of the specified meeting.
+
+Format: `editMeeting i/INDEX [d/DATE] [st/START TIME] [et/END TIME] [dn/DESCRIPTION]`
+
+* Edits the meeting at the specified `INDEX` (see [`delMeeting`](#deleting-a-meeting--delmeeting)).
```
Comment (rexong) [inline docs/UserGuide.md:282]: hmmm why do the user need to see the `delMeeting` command?

File: docs/UserGuide.md [hunk 5]
```diff
@@ -333,17 +366,20 @@ If your changes to the data file makes its format invalid, MyInsuRec will discar
...
-| **Delete meeting**    | `delMeeting i/INDEX` <br> e.g., <br> • `delMeeting i/1`                                                                                                            |
-| **Exit**              | `exit`                                                                                                                                                             |
+<<<<<<< HEAD
+| Action                | Format, Examples                                                                                                                                                                                         |
+|-----------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Add client**        | `addClient n/NAME p/PHONE_NUMBER [a/ADDRESS] [e/EMAIL] [b/BIRTHDAY] [pd/PRODUCT]` <br> e.g., <br> • `addClient n/John Tan p/0123456789` <br> • `addClient n/John Tan p/0123456789 e/johntan@insurec.com` |
```
Comment (rexong) [inline docs/UserGuide.md:372]: this one probably have some conflict to resolve. do take a look!

**Review by rexong — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 1 above)
Comment (Thing1Thing2) [reply docs/UserGuide.md:130]: There is an error here, addClient doesn't take prefix `i/`, so going to change that part.

File: docs/UserGuide.md (hunk 5 above)
Comment (Thing1Thing2) [reply docs/UserGuide.md:372]: my bad

**Comment by Thing1Thing2** [comment]
Too many CI errors, creating new PR

**Pull request closed without merging**